docs/modules/reference/pages/provisioning/handlers/requisition-plugin.adoc

```diff
@@ -0,0 +1,7 @@
+
+[[requisition-plugin]]
+= Requisition Plugins
+
+{page-component-title} provides plugins that can provision nodes from additional platforms, including software-defined networking, hypervisor, and configuration management database (CMDB) platforms.
+
+Each of these plugins has a handler that invokes the plugin with default settings, and imports requisitions generated by that plugin.
```

Comment:
Each of these plugins has a handler that invokes the plugin with default settings.
Some plugins accept options in the form of query parameters, which shape the requisitions generated by that plugin and imported by Provisiond.